### PR TITLE
Replace Cliqz references with Ghostery

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Cliqz' adblocker is a JavaScript library for *blocking ads, trackers, and annoya
...
+Ghostery' adblocker is the easiest and most efficient way to block ads and trackers in your project. Only a few lines of code are required to integrate smoothly with [Puppeteer](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-puppeteer-example), [Electron](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-electron-example), a  Chrome- and Firefox-compatible [browser extension](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-webextension-example), or any environment supporting [JavaScript](https://github.com/ghostery/adblocker/tree/master/packages/adblocker) (e.g. Node.js or React Native).
```
Comment (philipp-classen) [inline README.md:53]:
```suggestion
Ghostery's adblocker is the easiest and most efficient way to block ads and trackers in your project. Only a few lines of code are required to integrate smoothly with [Puppeteer](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-puppeteer-example), [Electron](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-electron-example), a  Chrome- and Firefox-compatible [browser extension](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-webextension-example), or any environment supporting [JavaScript](https://github.com/ghostery/adblocker/tree/master/packages/adblocker) (e.g. Node.js or React Native).
```

Comment (philipp-classen) [reply README.md:53]: Or alternatively, "The Ghostery adblocker"

File: packages/adblocker-content/README.md [hunk 2]
```diff
@@ -31,8 +31,8 @@
   <br/>
   <a href="#badge">
     <img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square"></a>
-  <a href="https://twitter.com/cliqz">
-    <img alt="Follow Cliqz on Twitter" src="https://img.shields.io/twitter/follow/cliqz.svg?label=follow+cliqz&style=flat-square"></a>
+  <a href="https://twitter.com/ghostery">
```
Comment (philipp-classen) [inline packages/adblocker-content/README.md:34]:
```suggestion
  <a href="https://x.com/ghostery">
```

File: packages/adblocker/tools/stress-test-engine-update.ts [hunk 3]
```diff
@@ -168,7 +168,7 @@ async function getMeta(url: string): Promise<any> {
  * Return the CDN URL of a specific revision `rev` (a hash) of subscription `list` (e.g. 'easylist').
  */
 function urlOfRevision(list: string, rev: string): string {
-  return `https://cdn.cliqz.com/adblocker/resources/${list}/${rev}/list.txt`;
+  return `https://cdn.ghostery.com/adblocker/resources/${list}/${rev}/list.txt`;
```
Comment (philipp-classen) [inline packages/adblocker/tools/stress-test-engine-update.ts:171]: Is this intended?

File: packages/adblocker/tools/stress-test-engine-update.ts [hunk 4]
```diff
@@ -269,7 +269,7 @@ async function collectTestCases(list: string): Promise<TestCase[]> {
     currentRevision: string;
     previousRevision: string;
   }[] = [];
-  const meta = await getMeta(`https://cdn.cliqz.com/adblocker/resources/${list}/metadata.json`);
+  const meta = await getMeta(`https://cdn.ghostery.com/adblocker/resources/${list}/metadata.json`);
```
Comment (philipp-classen) [inline packages/adblocker/tools/stress-test-engine-update.ts:272]: Is this intended?

File: README.md [hunk 5]
```diff
@@ -41,7 +41,7 @@
 
 ---
 
-Cliqz' adblocker is a JavaScript library for *blocking ads, trackers, and annoyances* with a strong focus on [efficiency](https://whotracks.me/blog/adblockers_performance_study.html). It was designed with compatibility in mind and integrates seamlessly with the following environments:
+Ghostery' adblocker is a JavaScript library for *blocking ads, trackers, and annoyances* with a strong focus on [efficiency](https://whotracks.me/blog/adblockers_performance_study.html). It was designed with compatibility in mind and integrates seamlessly with the following environments:
```
Comment (philipp-classen) [inline README.md:44]: (same as below)

File: packages/adblocker/tools/stress-test-engine-update.ts (hunk 3 above)
Comment (philipp-classen) [reply packages/adblocker/tools/stress-test-engine-update.ts:171]: I think, the URL is valid. So, it should be okay.

File: packages/adblocker/tools/stress-test-engine-update.ts (hunk 4 above)
Comment (philipp-classen) [reply packages/adblocker/tools/stress-test-engine-update.ts:272]: (as above)

File: packages/adblocker/tools/stress-test-engine-update.ts (hunk 3 above)
Comment (chrmod) [reply packages/adblocker/tools/stress-test-engine-update.ts:171]: cliqz cdn is no longer updated

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**